src/lib.rs

```diff
@@ -1,43 +1,7 @@
 #![deny(missing_docs)]
-#![doc(html_root_url = "https://docs.rs/fern/0.6.2")]
+#![doc(html_root_url = "https://docs.rs/fern/0.7.0")]
```

Comment:
Crates.io needs an external request to show the docs.rs link if it's not given in the Cargo.toml. It didn't show up for me for the longest time since my uMatrix blocked said request. Arguably my fault but you can still see it taking a split second longer to show up even when it works as expected.
There's currently no version number in the `documentation` link (nor should there be one), so no need to update it.
IMO it should stay as it doesn't really require maintenance.